server/server-kv.c

```diff
@@ -89,6 +140,9 @@ server_main(int sockfd, char* thread_number)
                 addr_size = sizeof client_addr;
                 clientfd = accept(sockfd, (struct sockaddr *)&client_addr, &addr_size);
                 //sh_print_client_ip(client_addr);
+		#ifdef DEBUG
```

Comment:
There is already a global flag for verbose printing in the works. See d093778c03580864141f29e48adefa6c509c8e3d